src/main/java/net/mcreator/element/converter/v2023_4/RangedItemToProjectileAndItemConverter.java

```diff
@@ -116,10 +116,13 @@ else if (rangedItem.get("ammoItem") != null && !rangedItem.get("ammoItem").getAs
 			item.name = rangedItem.get("name").getAsString();
 			item.texture = rangedItem.get("texture").getAsString();
 			item.renderType = rangedItem.get("renderType").getAsInt();
-			if (rangedItem.get("customModelName") != null)
+			if (rangedItem.get("customModelName") != null) {
 				item.customModelName = rangedItem.get("customModelName").getAsString();
-			if (item.customModelName.equals("Normal") && item.renderType == 0)
-				item.customModelName = "Ranged item"; // Convert the ranged items' normal model to items' RI model option
+				if (item.customModelName.equals("Normal") && item.renderType == 0)
+					item.customModelName = "Ranged item"; // Convert the ranged items' normal model to items' RI model option
+			} else {
+				item.customModelName = "Ranged item";

```

Comment:
Should there be another model assigned?

Reply:
If nothing is defined, ranged item should be assumed

Reply:
So is it fine or not?

Reply:
I think it is fine

Reply:
My bad, in pull request diff it looked like these two assignments were on the same level.